Eevee.Sleep.Bot/Handlers/InteractionHandler.cs

```diff
@@ -116,4 +119,25 @@ await interaction
                 throw new ArgumentException($"Unhandled modal ID: {modalId}");
         }
     }
+
+    private static async Task OnButtonClicked(SocketMessageComponent component)
+    {
+        var customId = component.Data.CustomId;
+        if (!customId.StartsWith("role"))
```

Comment:
Refactor to enum (refer to `ModalId` to similar impl for easier refactoring in the future.

Reply:
Fixed to use ButtonId
[042bf5a](https://github.com/RaenonX-PokemonSleep/eevee-sleep-discord-bot/pull/25/commits/042bf5ae19def3a76b4fa219af2021bdffdd459e)